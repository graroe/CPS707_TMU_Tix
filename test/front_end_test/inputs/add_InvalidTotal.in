login
admin
add
CirqueDuSoliel
9999
!q
logout