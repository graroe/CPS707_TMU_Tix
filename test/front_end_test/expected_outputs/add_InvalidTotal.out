Enter session type <sales or admin>: 
Admin login successful.
Enter event name: 
Event Date: 2023/10/27\nCurrent number of tickets: 10\nEnter number of tickets to add: 
Cannot exceed 9999 tickets. \nEnter number of tickets to add: 
Transaction canceled
Session terminated.